src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography;
```

Comment:
>Cryptography [](http://example.com/codeflow?start=22&length=12)

this is strange to see in a rename pr?

Reply:
It is indeed. That shouldn't have been added.